ec2/spark_ec2.py

```
@@ -983,6 +969,12 @@ def real_main():
     (opts, action, cluster_name) = parse_args()
 
     # Input parameter validation
+    if "." in opts.spark_version:
```

+1 on failing fast here